tailwind.config.js

```diff
@@ -285,18 +288,15 @@ module.exports = {
       ]
     },
     fontSize: {
-      0: '11px',
-      xs: '16px',
-      sm: '20px',
-      base: '24px',
-      lg: '32px',
-      xl: '40px',
-      '2xl': '1.5rem',
-      '3xl': '1.875rem',
-      '4xl': '2.25rem',
-      '5xl': '3rem',
-      '6xl': '4rem',
-      'xxs': '0.65rem',
+      xs: '11px',
```

Comment:
This naming convention is loads clearer and also it doesn't tie you to implementation because you can change the values without renaming the variables wherever they are used.

Reply:
I'm going to add a card to the backlog to look at this again later down the line. I haven't used tailwind or this component library enough yet so I think changing it might pop up a surprise and delay progress. Once we have the first version of the site up we can look at better naming conventions. Shouldn't be too bad to swap them out...

Reply:
https://tailwindcss.com/docs/configuration#theme 
I think we can change it, the theme configuration is completely up to us. On the other hand - we use a skeleventy boilerplate. So if any issues would occur in the future - it would be from Skeleventy, but not Tailwind.